packages/storybook/CONTRIBUTING.md

```diff
@@ -79,6 +83,8 @@ Note: if linking in a story via a native or Nimble anchor component, use the fol
 <a href="./?path=/docs/some--id" target="_top">Link</a>
 ```
 
+Linking to headings within a document doesn't work very well; avoid using links and refer to the section using **Bold** instead.
```

Comment:
Wasn't very clear to me standalone, tried to add examples in it, feel free to tweak

```suggestion
Linking to headings within a document doesn't work very well, i.e. `./page#some-heading`; avoid using links to specific headings and instead link to the page and refer to the section using **Bold**, i.e. `See **Some Heading** on [Page](./page)`.
```